vhs-frontend/src/components/SearchBar.tsx

```diff
@@ -1,49 +1,52 @@
-import { VHS } from "../types";
+import { FormEvent, useState, ChangeEvent } from 'react';
 
-type Props = {
-  searchBase: VHS[];
-  setFilteredList: Function;
-  setSearching: Function;
-};
+interface Props {
+  setQuery: (value: string) => void;
+}
+export const SearchBar = ({ setQuery }: Props) => {
+  const [inputVal, setInputVal] = useState('');
+  const [queryType, setQueryType] = useState('title');
```

Comment:
It would be good to give this `useState` a generic type that constrains it to only three possible values: `'title'`, `'description'` or `'genre'`.

This is how you would defines such a type: `export type QueryType = 'title' | 'description' | 'genre'`.

We call these kind of types union types because they use the pipe `|` operator.